packages/integration/lib/src/widgets/lists/feed/feed_list.dart

```diff
@@ -405,6 +406,7 @@ class _LMFeedListState extends State<LMFeedList> {
             LMFeedFireAnalyticsEvent(
               eventName: LMFeedAnalyticsKeys.postLiked,
               deprecatedEventName: LMFeedAnalyticsKeysDep.postLiked,
+              widgetSource: LMFeedWidgetSource.universalFeed,
```

Comment:
same for here is it right to assign widgetSurce: LMFeedWidgetSource.universalFeed ?